viadot/flows/adls_to_azure_sql.py

```diff
@@ -82,6 +82,7 @@ def df_to_csv_task(df, remove_tab, path: str, sep: str = "\t"):
 def check_dtypes_sort(
```

Comment:
Modify 'calling' this function in line no 288 (add new argument).

Reply:
done